tests/TestCase/I18n/DateTest.php

```diff
@@ -112,6 +112,16 @@ public function testI18nFormat($class)
 
         $result = $time->i18nFormat(\IntlDateFormatter::FULL, null, 'es-ES');
         $this->assertContains('14 de enero de 2010', $result, 'Default locale should not be used');
+
+        $time->setToStringTimezone('Europe/Brussels');
+        $result = $time->i18nFormat('HH:mm:ss');
+        $expected = '14:59:28';
+        $this->assertEquals($expected, $result);
```

Comment:
this tests can't pass with `Cake\I18n\Date` or with `Cake\I18n\FrozenDate` as the time part is always 00:00:00